README.md

```diff
@@ -17,14 +17,15 @@ We hope that allRank will facilitate both research in neural LTR and its industr
 ## Features
 
 ### Implemented loss functions:
- 1. ListNet (For a binary and graded relevance)
+ 1. ListNet (for binary and graded relevance)
  2. ListMLE
  3. RankNet
  4. Ordinal loss
  5. LambdaRank
  6. LambdaLoss
  7. ApproxNDCG
  8. RMSE
+ 9. NeuralNDCG (introduced in https://arxiv.org/pdf/2102.07831)
```

Comment:
you could also add a note about NeuralNDCG paper to the Research section of readme

Reply:
Added!